plugins/plugin-github/che-plugin-github-factory-resolver/src/main/java/org/eclipse/che/plugin/github/factory/resolver/GithubUrl.java

```diff
@@ -173,6 +173,21 @@ protected String devfileFileLocation() {
         .toString();
   }
 
+  /**
+   * Provides location to raw content of specified file
+   *
+   * @return location of specified file in a repository
+   */
+  protected String rawFileLocation(String fileName) {
+    return new StringJoiner("/")
```

Comment:
This code duplicates `protected String devfileFileLocation()`. I think it would be better to avoid such duplication by rewriting `devfileFileLocation` to 
```java
protected String devfileFileLocation() {
  return rawFileLocation(devfileFilename);
}
```

Reply:
Ok